README.rst

```diff
@@ -1,33 +1,19 @@
 Overall Status
 --------------
 
-+----------------------+------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
What's your justification for this removal?
Note that this README is only present on GitHub. It is not on the PyPI pages (use the API-specific READMEs) or ReadTheDocs sites (use the index.rsts which will look somewhat similar to the api-specific READMEs).

Reply:
>The docs link points to https://nimi-python.readthedocs.io/ which is outdated

Fair.

>The coverage status says that it's hasn't been run since 2020?

Yikes. That's because of #1218 . We should update the link to a codecov link, instead.
https://codecov.io/github/ni/nimi-python/graph/badge.svg